src/components/atoms/button/button.html

```diff
@@ -1,7 +1,3 @@
-<body>
-  <div class="text-padding">
-    <div class="text">
-      <a href="#">テキスト</a>
-    </div>
+ <div class="text-padding">
+    <a href="#">テキスト</a>
```

Comment:
```suggestion
 <div class="button">
    <a href="#">テキスト</a>
```
`text-padidng`というのは汎用的なクラス名なので、別の箇所とバッティングする可能性があると思います。
今回作成するパーツは `button` なので、ひと目でそれとわかり、他とかぶりにくいクラス名にしましょう。
かぶりにくさを重視すると`.atoms-button`としてもいいかもですね